app/views/settings/ext_cert/index.html.slim

```diff
@@ -0,0 +1,138 @@
+= render 'settings/apply'
+
+h2 External Certificates
+
+p Enable the use of External SSL Certificates for CaaSP and Kubernetes services.  Certificates and matching keys must be in armored PEM format.
+.alert.alert-warning role='alert'
+ p Remember to add a Root CA Certificate to the System Wide Certificates corresponding to any new External Certificates
+
+= form_for :external_certificate, url: settings_ext_cert_index_path, method: :post do |f|
+  br
+  .panel.panel-default
+    .panel-heading
+      h3.panel-title Velum SSL Certificate and Key
+    .panel-body
+      - if @velum_cert
+        .cert-table
+          h5 Current Velum SSL Certificate Info:
+          table
+            - @velum_cert.each do |k, v|
+              tr
+                td.field = k.to_s
+                td.value = v.to_s
+      - else
+        .cert-table
+          p
+          | No Velum certificates found.  Please add a certificate.
```

Comment:
You can simply write `p No Velum certificates found.  Please add a certificate.`. Also applies to the other occurrences.

Reply:
Done